### ISSUE TITLE
Non-text content accessibility improvements

### ISSUE THREAD

**Issue by trebuan202** [gchq/stroom#5398]
1.1.1 Non-Text Content (Partially Supports)

Within a Stroom Pipeline, each pipeline element contains an icon to highlight what type of element they represent ( e.g. filter, XSL etc ), however, there is no textual alternative for this icon. The screen reader therefore is unable to give any textual representation of this icon.
